src/Nav.js

```diff
@@ -0,0 +1,18 @@
+// #4 This is our new Nav component
```

Comment:
You'll need to import React in this file to avoid the "'React' must be in scope when using JSX" bug in your console. Your app fails to compile for me without it.